packages/runtime/src/api/createAPIv2.ts

```diff
@@ -80,7 +80,7 @@ export function createAPI(
     const evaluatedInputs = Object.entries(api.inputs).reduce<
       Record<string, unknown>
     >((acc, [key, value]) => {
-      acc[key] = applyFormula(value.formula, formulaContext)
+      acc[key] = applyFormula(value.formula ?? undefined, formulaContext)
```

Comment:
We usually try to avoid `undefined` in the editor. Would it make sense to use `null` instead here?
```suggestion
      acc[key] = applyFormula(value.formula ?? null, formulaContext)
```